### DISCUSSION TITLE
ubuntu下frpc客户端怎么替换 SSL 证书文件

### DISCUSSION THREAD

**Discussion by anran2020** [natfrp/wiki#262] [使用问题] [answered]
求解：ubuntu下frpc客户端怎么替换 SSL 证书文件

https隧道，和域名绑定均已配置好，并且申请了对应域名的ssl证书

可从外网通过https://api.example.com访问网站，但是会提示“此连接非私人连接”，点击显示隐藏信息后，查看证书显示不匹配，要想访问网站还得点击“访问此网站”，才跳转

在教程中貌似只看到了windows下替换ssl证书的方法，但是ubuntu下如何替换sakura frp自签证书替换成自己申请的域名ssl证书

**Comment by anran2020** [comment]
![image](https://user-images.githubusercontent.com/61401619/212466016-67d0666f-4bf6-42b0-a69e-ca1e1564e341.png)

**Comment by anran2020** [comment]
如果在nginx.conf中配置了申请的ssl证书相关的信息，并打开ssl，此时通过https://example.com访问网站会出现重定向错误
![image](https://user-images.githubusercontent.com/61401619/212466239-fdcc6eb7-f6ad-4082-b33b-26cceb8e1d25.png)

**Comment by fakeboboliu** [comment, marked as answer]
如果在 web server 中配置了 https，则请关闭 `自动 https`，并对应修改隧道设置的本地端口

**Reply by anran2020** [reply]
我在自动https中填写了绑定的域名

**Comment by anran2020** [comment]
> 求解：ubuntu下frpc客户端怎么替换 SSL 证书文件
> 
> 
> 
> https隧道，和域名绑定均已配置好，并且申请了对应域名的ssl证书
> 
> 
> 
> 可从外网通过https://api.example.com访问网站，但是会提示“此连接非私人连接”，点击显示隐藏信息后，查看证书显示不匹配，要想访问网站还得点击“访问此网站”，才跳转
> 
> 
> 
> 在教程中貌似只看到了windows下替换ssl证书的方法，但是ubuntu下如何替换sakura frp自签证书替换成自己申请的域名ssl证书

还是这个问题困扰着哈哈

**Reply by fakeboboliu** [reply]
> 关闭(置空) 自动 https，并对应修改隧道设置的本地端口

**Reply by anran2020** [reply]
非常感谢回复，我先试试

这个证书不匹配的问题感觉像是自签证书没有没被替换成我申请的证书
